cmip6_downscaling/data/cmip.py

```diff
@@ -220,19 +220,48 @@ def load_cmip_dictionary(
     elif return_type == 'xr':
         ds = xr.open_zarr(stores[0], consolidated=True)
 
+    # flip the lats if necessary and drop the extra dims/vars like bnds
+
     ds = gcm_munge(ds)
 
     return ds
 
 
-def convert_to_360(lon):
+def convert_to_360(lon: Union[float, int]) -> Union[float, int]:
```

Comment:
typing comments:
- Do we expect to ever pass integers in here?
- I suspect mypy will complain when you pass a array-like type in here